tests/publiccloud/check_registercloudguest.pm

```diff
@@ -106,14 +105,16 @@ sub run {
     if (is_byos()) {
         $instance->ssh_assert_script_run(cmd => 'sudo SUSEConnect --version');
         $instance->ssh_assert_script_run(cmd => "sudo SUSEConnect $regcode_param");
-        $instance->ssh_assert_script_run(cmd => "sudo ${path}registercloudguest --clean");
+        cleanup_instance($instance);
     }
 
-    $instance->ssh_script_retry(cmd => "sudo ${path}registercloudguest $regcode_param", timeout => 300, retry => 3, delay => 120);
-    check_instance_registered($instance);
+    new_registration($instance);
 
-    $instance->ssh_script_retry(cmd => "sudo ${path}registercloudguest $regcode_param --force-new", timeout => 300, retry => 3, delay => 120);
-    check_instance_registered($instance);
+    if (is_sle('>=15')) {
+        test_container_runtimes($instance);
+    }
+
+    force_new_registration($instance);
```

Comment:
same comment as https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20260/files#r1780879917

Reply:
same here scenario of forcing new registration should working exactly the same on PAYG instance and registered BYOS instance

Reply:
ok